static/js/find-a-partner.js

```diff
@@ -0,0 +1,170 @@
+(function() {
+
+  var filters = [];
+
+  function init() {
+    revealSearch();
+
+    var inputNode = document.querySelector(".p-find-a-partner__search-input");
+    inputNode.addEventListener("keyup", searchHandler);
```

Comment:
You probably want to null check this as it will error if someone changes the class. Can you also use `js-` prefixed classes so its clear they apply functionality and no styling.